snapshots/un/2024-08-27/un_sdg_dimension.json.dvc

```diff
@@ -0,0 +1,29 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: United Nations Sustainable Development Goals - Dimensions
+    date_published: "2024"
```

Comment:
I think the date is 2024-06-28, based on this file https://unstats.un.org/sdgs/indicators/SDG_Updateinfo.xlsx

Reply:
Same for the other two dvc files

Reply:
Ah, nice, thanks!